prime-angular-frontend/src/app/modules/site-registration/shared/guards/organization.guard.ts

```diff
@@ -127,10 +130,14 @@ export class OrganizationGuard extends BaseGuard {
    * Manage routing when an organization does not exist, or initial
    * registration has not been completed.
    */
-  private manageNoOrganizationRouting(routePath: string) {
+  private manageNoOrganizationRouting(routePath: string, party: Party, hasOrgClaim: boolean) {
+    const destPath = hasOrgClaim ? SiteRoutes.ORGANIZATION_CLAIM_CONFIRMATION :
```

Comment:
When using ternaries unless they are really short they should always be structured vertically for readability:

```
const destPath = (claimed) 
  ? SiteRoutes.ORGANIZATION_CLAIM_CONFIRMATION 
  : (party) 
    ? SiteRoutes.ORGANIZATION_CLAIM 
    : SiteRoutes.ORGANIZATION_SIGNING_AUTHORITY;
```